README.md

```diff
@@ -19,5 +19,27 @@ The Span normalizer uses gradlew to compile/install/distribute. Gradle wrapper i
 ./gradlew dockerBuildImages
 ```
 
+## Testing
+
+### Running unit tests
+Run `./gradlew test` to execute unit tests. 
+
+
+### Testing image
+
+You can test the image you built after modification by running docker-compose or helm setup. 
+
+#### docker-compose
+Change the tag for `span-normalizer` from `:main` to `:test` in [docker-compose file](https://github.com/hypertrace/hypertrace/blob/main/docker/docker-compose.yml) like this.
```

Comment:
This is not generating `test` tag as auto for now in this branch.

Reply:
@JBAhire  Change this in  `/span-normalizer/build.gradle.kts`
``` 
id("org.hypertrace.docker-java-application-plugin") version "0.7.1"
id("org.hypertrace.docker-publish-plugin") version "0.7.1"
```

Reply:
done!

Reply:
once in the macrorepo we would make a docker/docker-compose section there so that testing is in the same repo is the code that is effected.